doc/source/examples/analysis/alignment_and_rms/rmsf.ipynb

```diff
@@ -18,8 +18,12 @@
     "* MDAnalysis (<a data-cite=\"michaud-agrawal_mdanalysis_2011\" href=\"https://doi.org/10.1002/jcc.21787\">Michaud-Agrawal *et al.*, 2011</a>, <a data-cite=\"gowers_mdanalysis_2016\" href=\"https://doi.org/10.25080/Majora-629e541a-00e\">Gowers *et al.*, 2016</a>)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import nglview as nv
</pre><p>Following from the above - it's not commented out here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
my mistake -- fixed